src/components/Button/Button.stories.tsx

```diff
@@ -83,43 +83,60 @@ SecondaryMedium.args = { children: 'Button', variant: 'secondary', size: 'md' };
 export const SecondarySmall = Template.bind({});
 SecondarySmall.args = { children: 'Button', variant: 'secondary', size: 'sm' };
 
-export const Tertiary = Template.bind({});
-Tertiary.args = { children: 'Button', variant: 'tertiary' };
+export const SecondaryNeutral = Template.bind({});
```

Comment:
I changed the story name from "Tertiary" to "SecondaryNeutral" so it's consistent with the props, but now it's at odds with the UI Kit. I can change it back if anyone feels strongly that it should match the UI Kit instead.

Reply:
hmmm I think it's helpful for designers when story names match the UI Kit (and eng can click "Show Code"). we should probably sync or pair with designers on naming though, because "Secondary" now has a different meaning in code than in the UI Kit

Reply:
I'll add a ticket to the backlog for syncing the names. I don't think it's time-sensitive but I feel like we keep coming back to this inconsistent naming issue.

If our concern is the code terms "primary" and "secondary" are conflicting with the UI Kit terms "primary" and "secondary", well, we could always go back to "flat" and "outline" in the code 🤷 

For now, I'll change this one to "Tertiary" and primary error to "Destructive", thanks!